dspace/config/spring/api/discovery.xml

```diff
@@ -53,9 +53,6 @@
                 <!-- Same as the "default" configuration, but does NOT filter out older versions of items -->
                 <!-- Used to display related items on single-item pages, because a relationship does not always point to the latest version of the related item -->
                 <entry key="default-relationships" value-ref="defaultRelationshipsConfiguration" />
-
-                <!--Use site to override the default configuration for the home page & default discovery page-->
-                <entry key="site" value-ref="homepageConfiguration" />
                 <!--<entry key="123456789/7621" value-ref="defaultConfiguration"/>-->
```

Comment:
why are we removing such option? it was supported in previous version and implemented on the rest side. This allow institution to add custom components in their home page that use a different configuration than the default search

Reply:
The homepage configuration is not being used in DSpace (both REST and Angular).
Both the search bar on the homepage and the search icon in the header go to the general "/search" page using the "default" configuration where you can also go back and forth between community/collection based scopes. 

I removed this configuration since it is confusing to have a configuration present that does not do anything.

Reply:
I agree that we should remove this as it's no longer used/needed.  It makes no sense to keep configurations in `discovery.xml` which are unused.